tests/test_limoss.py

```
@@ -0,0 +1,262 @@
+        mock_bleak_client.write_gatt_char.reset_mock()
+
+        await controller.move_feet_up()
+
+        first_packet = mock_bleak_client.write_gatt_char.call_args_list[0].args[1]
+        assert controller._decode_packet(first_packet)[0] == LimossCommands.MOTOR_4_UP
+
+    async def test_capability_response_updates_memory_slots(
+        self,
+        hass: HomeAssistant,
+        mock_limoss_config_entry,
+        mock_coordinator_connected,
```

Ruff (0.14.14)

[warning] Unused method argument: `mock_coordinator_connected`

(ARG002)